charms/openfga-k8s/charmcraft.yaml

```diff
@@ -6,13 +6,12 @@ parts:
   charm:
     charm-python-packages: 
       - setuptools
-      - pyopenssl
-    build-packages:
-      - pkg-config
-      - libffi-dev
-      - libssl-dev
-      - rustc
-      - cargo
+    charm-binary-python-packages:
```

Comment:
I thought we just wanted to use binary packages for the CI performance. Are we okay have them for building charms (release)?

Reply:
After asking the question, it seems there is nothing particularly risky about using it for the built charms as well.